src/vs/workbench/browser/parts/compositePart.ts

```diff
@@ -426,6 +428,8 @@ export abstract class CompositePart<T extends Composite> extends Part {
 			'class': ['composite', 'title']
 		});
 
+		this._register(DOM.addDisposableListener(titleArea.getHTMLElement(), DOM.EventType.CONTEXT_MENU, e => this.onContextMenu(new StandardMouseEvent(e))));
```

Comment:
@sandy081 it is a old helper class but if we use it elsewhere in the same file I think it is good to be consistent. One advantage of the builder is that any event listeners that were created using the builder are being disposed automatically once any of the dom nodes created with that builder get disposed.